smart-contracts/test/Lock/grantKeys.js

```diff
@@ -1,5 +1,6 @@
 const Web3Utils = require('web3-utils')
 
+const truffleAssert = require('truffle-assertions')
```

Comment:
is this new as of this PR as well?

Reply:
https://github.com/unlock-protocol/unlock/pull/4784